### ISSUE TITLE
Set semseg  as logit or probability?

### ISSUE THREAD

**Issue by haozhi1817** [zbwxp/SegVit#15]
https://github.com/zbwxp/SegVit/blob/85616388cd8983e2d4daafd971540b210e956666/decode_heads/atm_single_head.py#L277
semseg = mask_cls.softmax(-1) @ mask_pred.sigmoid()
which will result in: **1>= semseg >= 0**
When we compute [loss_func](https://github.com/zbwxp/SegVit/blob/85616388cd8983e2d4daafd971540b210e956666/losses/criterion.py#L58), we set semseg as a **logits**, and compute prob by semseg.sigmoid()
If **1>= semseg >= 0**，the probs must be >= 0.5,  but we need the prob >=0.
Maybe I am wrong or I miss some calculation process，please feel free to advise me。
